hyperlink/test/test_scheme_registration.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+from .common import HyperlinkTestCase
+from .._url import register_scheme, URL
+
+
+class TestSchemeRegistration(HyperlinkTestCase):
+
+    def test_register_scheme_basic(self):
+        register_scheme('deltron', uses_netloc=True, default_port=3030)
```

Comment:
Looking at the sheer prevalence of use, I think use_netloc=True is a more sensible default and would meet more expectations. I'll remove the option of having `None`, it's not saving anyone a boatload of typing or anything. Sound good?